docs/features.md

```diff
@@ -119,12 +120,17 @@ Format your code with various Haskell code formatters.
 | Ormolu          | `hls-ormolu-plugin`          |
 | Stylish Haskell | `hls-stylish-haskell-plugin` |
 
+---
```

Comment:
does this do anything?

Reply:
I think this markdown syntax is usually translated to `<hr/>` in html, so should be fine.